core-ui/src/main/java/com/anytypeio/anytype/core_ui/features/editor/holders/media/File.kt

```diff
@@ -84,6 +86,18 @@ class File(val binding: ItemBlockFileBinding) : Media(binding.root), Decoratable
         applyBackground(item.background)
     }
 
+    private fun setEllipsizedText(spannable: Spannable) {
+        name.post {
```

Comment:
need some clarifications here: what happens for instance if you don't use `post {...}` block ?

Reply:
Without post, the code may run before TextView has its width, causing file names not to display correctly.

Reply:
It seems that you only need to set the text in the post { name.text =  }